spec/var-sub.test.sh

```diff
@@ -19,6 +19,11 @@ echo ${foo:-$({ which ls; })}
 # BUG bash stdout-json: ""
 # BUG bash status: 2
 
+### :- with empty alternative
+foo=
+argv.py "${foo:-}"
+# stdout: ['']
+
```

Comment:
OK wow I'm surprised this is broken!!  Can you put this test at the very top of spec/var-sub-quote.test.sh?

Then I will pull it.  Thanks.

Reply:
Done!

Reply:
OK can you push it or do I have to pull?  I haven't used this "approve" feature by Github before.  I would assume it should non-committers push one change?

Reply:
Don't think so, it just marks approval. I think you can set it up to e.g. automatically merge once it has n approvals, but it doesn't allow the PR author to merge it themselves.

Reply:
OK but I think you need to push the update you made?  I still see 3 commits and the change in `var-sub` not `var-sub-quote`.  I would like it to allow pushing so it doesn't require another round trip :)

Reply:
Whoops, sorry, misread — I didn't realise you were referring to the other script. Fixed!